rector.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Rector\Config\RectorConfig;
+use Rector\DeadCode\Rector\ClassLike\RemoveAnnotationRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->paths([
+        __DIR__ . '/examples',
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+        __DIR__ . '/tools',
+    ]);
+
+    /**
+     * All classes are public API by default, unless marked with @internal.
+     */
+    $rectorConfig->ruleWithConfiguration(RemoveAnnotationRector::class, ['api']);
```

Comment:
I don't think PHPLIB does this, but is there any harm of removing `@api` from method annotations as well?

Reply:
The purpose of leaving these rules in place in the configuration file is to impose them on new developments or pending branches. If, for example, a developer forgets or ignores that `@api` annotations are no longer used, this will be corrected automatically before the code review.